configure

```diff
@@ -1156,6 +1157,9 @@ Saved environment variables:
 AR="$SAVED_AR" AS="$SAVED_AS" ASFLAGS="$SAVED_ASFLAGS" CPPFLAGS="$SAVED_CPPFLAGS" CXX="$SAVED_CXX" CXXFLAGS="$SAVED_CXXFLAGS" DWP="$SAVED_DWP" LD="$SAVED_LD" LDFLAGS="$SAVED_LDFLAGS" PKG_CONFIG_LIBDIR="$SAVED_PKG_CONFIG_LIBDIR" RANLIB="$SAVED_RANLIB" SDL_CONFIG="$SAVED_SDL_CONFIG" STRIP="$SAVED_STRIP" WINDRES="$SAVED_WINDRES" WINDRESFLAGS="$SAVED_WINDRESFLAGS"
 EOF
 
+JPEG_LIBS=-ljpeg
```

Comment:
This changes the defaults, whatever they are. Say, you have it in your env variables. Is this intentional?

Reply:
it's to compensate for later code not adding -ljpeg anymore. As a possibility I can switch it to "$JPEG_LIBS -ljpeg"  for it to mimic old behaviour but this behaviour is undocumented and --with-jpeg-prefix is the documented way.